apps/dash-svm/requirements.txt

```diff
@@ -0,0 +1,15 @@
+# Core
+gunicorn>=19.8.1
+plotly==2.7.0
+dash>=0.40.0
+dash-renderer>=0.21.0
```

Comment:
You don't need renderer, html, or core any more (they're auto-installed with Dash).